examples/vision/cifar10.exs

```diff
@@ -14,9 +14,10 @@ defmodule Cifar do
   defp transform_images({bin, type, shape}) do
     bin
     |> Nx.from_binary(type)
-    |> Nx.reshape({elem(shape, 0), 3, 32, 32})
+    |> Nx.reshape({elem(shape, 0), 3, 32, 32}, names: [:n, :c, :h, :w])
```

Comment:
```suggestion
    |> Nx.reshape(shape, names: [:n, :c, :h, :w])
```

I think shape is already this way?

Reply:
Hey @grzuy thanks for your comment!

Honestly I don't remember anymore, I should run the livebook to check it 🙈